packages/lib-classifier/src/store/Step.js

```diff
@@ -0,0 +1,9 @@
+import { types } from 'mobx-state-tree'
+
+const Step = types.model('Step', {
+  next: types.optional(types.string, ''), // We have this optionally to support recursive workflows
+  stepKey: types.identifier,
```

Comment:
This is duplicating the key of the Map key-pair, but the model needs an identifier too I think? I'm not sure. This example shows a separate identifier from the key of the key-value pair of the Map: https://github.com/mobxjs/mobx-state-tree/blob/master/API.md#typesmap

Reply:
I think I'm doing this right after a reread of the docs.